presidio-analyzer/presidio_analyzer/predefined_recognizers/in_passport_recognizer.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional, List, Tuple
+
+from presidio_analyzer import Pattern, PatternRecognizer
+
+class InPassportRecognizer(PatternRecognizer):
+    """
+    Recognizes Indian Passport Number.
```

Comment:
Can you please reference the logic used to create this regex pattern?

Reply:
Let me know if below reference link works, all other reference link has the same information as given in the below link

https://www.bajajallianz.com/blog/travel-insurance-articles/where-is-passport-number-in-indian-passport.html#:~:text=What%20is%20the%20Passport%20Number%3F

Reply:
Yes, that's great, thanks!

Reply:
Reference Added.
Thank you